go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -22,6 +21,10 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
+
```

Comment:
nit: this blank line is unnecessary

Reply:
fixed, I had thought gofmt used to reorganize the imports canonically ...

Reply:
there are a wide variety of, uh, "arrangements" that `gofmt` accepts, so there isn't technically a "canonically".

for example, each of these are valid:

```go
import (
	"context"
	"flag"
	"fmt"
	"io"
	"sync"
	"time"

	"google.golang.org/grpc"

	"vitess.io/vitess/go/netutil"
	"vitess.io/vitess/go/vt/callerid"
	"vitess.io/vitess/go/vt/grpcclient"
	"vitess.io/vitess/go/vt/hook"
	"vitess.io/vitess/go/vt/log"
	"vitess.io/vitess/go/vt/logutil"
	"vitess.io/vitess/go/vt/mysqlctl/tmutils"
	"vitess.io/vitess/go/vt/topo/topoproto"
	"vitess.io/vitess/go/vt/vttablet/tmclient"
)
```

```go
import (
	"context"
	"io"
	"sync"

	"vitess.io/vitess/go/vt/log"

	"flag"

	"vitess.io/vitess/go/vt/callerid"

	"fmt"

	"google.golang.org/grpc"

	"vitess.io/vitess/go/netutil"
	"vitess.io/vitess/go/vt/grpcclient"
	"vitess.io/vitess/go/vt/hook"

	"time"

	"vitess.io/vitess/go/vt/logutil"
	"vitess.io/vitess/go/vt/mysqlctl/tmutils"
	"vitess.io/vitess/go/vt/topo/topoproto"
	"vitess.io/vitess/go/vt/vttablet/tmclient"
)
```

```go
import (
	"context"

	"github.com/spf13/pflag"

	"vitess.io/vitess/go/vt/mysqlctl/tmutils"

	"io"

	"vitess.io/vitess/go/vt/topo/topoproto"

	"sync"

	"vitess.io/vitess/go/vt/log"

	"flag"

	"vitess.io/vitess/go/vt/callerid"

	"fmt"

	"google.golang.org/grpc"

	"vitess.io/vitess/go/netutil"
	"vitess.io/vitess/go/vt/grpcclient"
	"vitess.io/vitess/go/vt/hook"

	"time"

	"vitess.io/vitess/go/vt/logutil"
	"vitess.io/vitess/go/vt/vttablet/tmclient"
)
```

now, obviously, this is at least partly aesthetic preference, and i'm being at least a little ridiculous with the examples i came up, but nonetheless i hope you agree the first one is the most pleasing / easily-scannable